packages/server/src/common/server.ts

```diff
@@ -47,22 +48,6 @@ export default class ExpressServer {
         app.use(express.static(`${root}/public`));
         app.use(express.static(`${clientRoot}/build`));
 
-        const apiSpec = path.join(__dirname, '../docs/api.yml');
```

Comment:
moved this below to `mountExplorer` which is called in `server/index.ts` instead of being mounted on creation. I was having issues with `YAML.load` while testing so decided to do it this way for now. Regardless, I think we need to clean up how the server is setup and exported.

Reply:
Yep, makes sense. This structure comes from a boilerplate and I've already torn some of that scaffolding down. Not averse to a new scaffolding flow.